### DISCUSSION TITLE
The unit of directed_velocity in PICMI inputs

### DISCUSSION THREAD

**Discussion by tanweihou** [BLAST-WarpX/warpx#4737] [Simulation Inputs] [answered]
Hello,

In the PICMI inputs such as [UniformDistribution](https://warpx.readthedocs.io/en/latest/usage/python.html#pywarpx.picmi.UniformDistribution), the unit of `directed_velocity` is listed as `m/s`.
However, in this [PWFA](https://warpx.readthedocs.io/en/latest/usage/examples/pwfa/README.html) example, the `directed_velocity` is `1e9` as shown in below
```python
beam_distribution = picmi.UniformDistribution(density = 1.e23,
                                              lower_bound = [-20.e-6, -20.e-6, -150.e-6],
                                              upper_bound = [+20.e-6, +20.e-6, -100.e-6],
                                              directed_velocity = [0., 0., 1.e9])
```
If the unit is `m/s`, it is already greater than the speed of light, isn't it?
Is there a typo?
Is the unit supposed to be $\gamma\beta$?

Thanks.

**Comment by ax3l** [comment, marked as answer]
Oh, that is because we mean [proper velocity](https://en.wikipedia.org/wiki/Proper_velocity), $\gamma(v) \cdot v$.

Let's fix that doc string in [PICMI](https://github.com/picmi-standard/picmi): https://github.com/picmi-standard/picmi/pull/109

**Reply by tanweihou** [reply]
Thanks!
